docs/installing.md

```diff
@@ -119,10 +119,16 @@ Helm defaults used in the `testkube` chart:
 | api-server.service.port | yes | 8088 |
 | api-server.mongoDSN | yes | "mongodb://testkube-mongodb:27017" |
 | api-server.telemetryDisabled | yes | false |
-| api-server.storage.endpoint | yes | localhost:9000 |
+| api-server.storage.endpoint | yes | testkube-minio-service-testkube:9000 |
 | api-server.storage.accessKeyId | yes | minio |
 | api-server.storage.accessKey | yes | minio123 |
 | api-server.storage.scrapperEnabled | yes | false |
 
+> Please note. Storage endpoint is built dinamically based on Minio's service namespace name.
+```
+testkube-minio-service-<minio_namespace_name>:9000
```

Comment:
can't we name this without minio namespace?

Reply:
We can. But then we will have to rename minio service as well. In a hardcoded manner.